src/commander/commander_btc.c

```diff
@@ -168,6 +168,20 @@ commander_error_t commander_btc_pub(const BTCPubRequest* request, PubResponse* r
     }
 }
 
+commander_error_t commander_btc_identifier_xpub(
+    const BTCIdentifierXPubRequest* request,
+    BTCIdentifierXPubResponse* response)
+{
+    if (!app_btc_identifier_xpub(
+            request->keypath,
+            sizeof(request->keypath) / sizeof(uint32_t),
```

Comment:
this must be `request->keypath_count`. the user can also send a `0` or `1` length keypath.

Reply:
Right, I should have asked you about this :/